src/main.cpp

```diff
@@ -469,6 +469,25 @@ struct Args {
   std::vector<std::string> debug_stages;
```

Comment:
nit: Should this be a set since we're potentially inserting the same stage twice?